data/json/dreams.json

```diff
@@ -639,14 +670,24 @@
     "type": "dream",
     "messages": [
       "You pant, terrified at the thought of that FUNGUS destroying your home!",
+      "Your first arrow hits the bullseye.  Your second arrow splits the first arrow in half.",
+      "You dance easily across a tightrope, never in any danger of falling.",
+      "You encourage your followers to be independent.  The more new ideas you try, the more chances that you'll find the best ideas!",
```

Comment:
These are creeping a bit too far into the fantasy elf trope.

Reply:
deleting these then